src/loaders/common/loader_common_names.py

```diff
@@ -61,3 +61,12 @@
 SOURCE_METADATA_FILE_KEYS = ["upa", "name", "type", "timestamp"]
 # callback server docker image name
 CALLBACK_IMAGE_NAME = "scanon/callback"
+
+# genome_attribs
+GENOME_ATTRIBS = "genome_attribs"
+
+DATA_ID_COLUMN_HEADER = "genome_id"  # TODO DATA_ID change to data ID for generality
```

Comment:
Updated!

Reply:
Looks ok, but let's call `META_DATA_ID` `data_id` or something like that that is more generic than `genome_id`, since this code will presumably dealing with more than genomes in the future

Reply:
:+1: